src/api-wrappers/AXUIElement.swift

```diff
@@ -247,6 +247,80 @@ extension AXUIElement {
     func performAction(_ action: String) {
         AXUIElementPerformAction(self, action as CFString)
     }
+    
+    private static let ignoredBundleIDs = Set([
+        "com.apple.dashboard",
+        "com.apple.loginwindow",
+        "com.apple.notificationcenterui",
+        "com.apple.wifi.WiFiAgent",
+        "com.apple.Spotlight",
+        "com.apple.systemuiserver",
+        "com.apple.dock",
+        "com.apple.AirPlayUIAgent",
+        "com.apple.dock.extra",
+        "com.apple.PowerChime",
+        "com.apple.WebKit.Networking",
+        "com.apple.WebKit.WebContent",
+        "com.apple.WebKit.GPU",
+        "com.apple.FollowUpUI",
+        "com.apple.controlcenter",
+        "com.apple.SoftwareUpdateNotificationManager",
+        "com.apple.TextInputMenuAgent",
+        "com.apple.TextInputSwitcher"
+    ])
```

Comment:
I don't think it's a good idea to hardcode specific `bundleIdentifier`s. If these start creating windows in further macOS releases, then AltTab will have to be updated to show them. Also how did you build that list? Maybe it's missing processes from previous macOS versions you don't use, or maybe (actually it's certain) different processes will run on your mac depending on your System Preferences, if you have a network printer, external monitor, keyboard, if your phone does AirPlay with the mac, etc.

I think we should find more flags or properties of apps that give strong hints that they will not spawn windows. Also retrying less often than 10ms is already a large workaround the CPU issue